app/gui2/src/util/database/reactiveDb.ts

```diff
@@ -326,11 +327,15 @@ export class ReactiveMapping<K, V, M> {
   constructor(db: ReactiveDb<K, V>, indexer: Mapper<K, V, M>, debugOptions?: DebuggerOptions) {
     this.computed = reactive(map.create())
     db.on('entryAdded', (key, value, onDelete) => {
-      this.computed.set(
-        key,
-        computed(() => indexer(key, value), debugOptions),
-      )
-      onDelete(() => this.computed.delete(key))
+      const scope = effectScope()
+      const mappedValue = scope.run(() =>
+        computed(() => scope.run(() => indexer(key, value)), debugOptions),
```

Comment:
I still think that this is somewhat dodgy to have a `scope.run` inside computed value. We will need to figure out a better way to support this usecase (probably a different db mapper type not based on computed). But for now let's merge it as is and check if this approach has any unintended consequences.